### ISSUE TITLE
Considerable Sdk Loading Time

### ISSUE THREAD

**Issue by marcoRS** [Instabug/Instabug-Android#215]
### Steps to Reproduce the Problem
Install versions 8.4.+, initialize the Sdk in Application class:
### Expected Behavior
The sdk loads within reasonable time no more than 1000 milliseconds
### Actual Behavior
App loading time on devices and emulator is more than 1000 and we get warnings of the form: `I/Choreographer: Skipped 92 frames!`

### Instabug integration code
```kotlin
Instabug.Builder(app, "MY_KEY_HERE")
            .setInvocationEvents(
                InstabugInvocationEvent.SHAKE,
                InstabugInvocationEvent.SCREENSHOT,
                InstabugInvocationEvent.TWO_FINGER_SWIPE_LEFT
            )
            .build()
```

### SDK Version
8.4.0, 8.5.0 , 8.6.0 
### Android Version
21+
### Device Model
Various emulators + devices
### Notes
I have tested the project with and without instabug and the difference is very noticeable. 

```
2019-08-07 15:07:32.926 3899-3944/org.myapp.debug V/FA: Connection attempt already in progress
2019-08-07 15:07:32.929 3899-3944/org.myapp.debug V/FA: Activity resumed, time: 74965900
2019-08-07 15:07:32.989 3899-3899/org.myapp.debug D/OpenGLRenderer: Skia GL Pipeline
2019-08-07 15:07:34.579 3899-3899/org.myapp.debug W/.flagship.debu: Long monitor contention with owner AsyncTask #1 (3992) at java.net.InetAddress[] libcore.io.Linux.android_getaddrinfo(java.lang.String, android.system.StructAddrinfo, int)(Linux.java:-2) waiters=0 in void com.instabug.library.b.c(android.content.Context) for 1.544s
2019-08-07 15:07:34.590 3899-3899/org.myapp.debug I/Choreographer: Skipped 94 frames!  The application may be doing too much work on its main thread.
2019-08-07 15:07:34.620 3899-3994/org.myapp.debug I/ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
2019-08-07 15:07:34.621 3899-3994/org.myapp.debug I/ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
2019-08-07 15:07:34.621 3899-3994/org.myapp.debug I/OpenGLRenderer: Initialized EGL, version 1.4
2019-08-07 15:07:34.621 3899-3994/org.myapp.debug D/OpenGLRenderer: Swap behavior 1
```

Theres something about this function: `in void com.instabug.library.b.c(android.content.Context) for 1.544s` That seems to be taking 1.54s to load.

**Comment by hanaamohamed** [comment]
Hello @marcoRS, 

you can try this snapshot for now, it has some improvement in launching time 
`'com.instabug.library:instabug:8.7.0-SNAPSHOT'`

In order to consume this snapshot you need to include the below in gradle:
```
      repositories {
          maven {
               url "https://oss.sonatype.org/content/repositories/snapshots"
           }
      }
 ```

**Comment by amrbrkt** [comment]
Hi @marcoRS,

This issue has been fixed and released in `8.6.3`. Please let us know if you still have any other issues.
